src/func-aeh-listener/src/functions/xxENSONOxx.xxSTACKSxx.Listener/Program.cs

```diff
@@ -15,14 +15,15 @@
     .ConfigureFunctionsWorkerDefaults()
     .ConfigureServices((context, services) =>
     {
+        services.AddSingleton<IMessageReader, JsonMessageSerializer>();
+
         var configuration = new ConfigurationBuilder()
             .SetBasePath(context.HostingEnvironment.ContentRootPath)
-            .AddJsonFile("appsettings.json", false)
+            .AddJsonFile("local.settings.json", false)
```

Comment:
Might it be best to keep this as the appSettings.json file as local.settings.json is used for running locally?

Reply:
appSettings isn't typically used with an azure function, it's a webapi thing

Reply:
I think the settings in local.settings.json will only get used in local dev environments, when the function is published, it gets ignored.  I've not tried it in a while though.  Just had a quick look in the docs and [found this](https://learn.microsoft.com/en-us/azure/azure-functions/functions-develop-local#local-settings-file), my interpretation is that it will still get ignored.

I agree though,  I've not seen an appsettings.json in a functions app before these projects.